2.1/slidingWindow_test.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"sort"
+	"testing"
+)
+
+//test setup
+var (
+	seed     int64 = 1
+	testRand       = rand.New(rand.NewSource(seed))
+)
+
+func TestRecordValue(t *testing.T) {
+	r := &R{data: seedData([]uint{0, 1, 2, 3}), percentile: .95, window: 100}
+	expectedLength := len(r.data) + 1
+	r.RecordValue(101)
+	if len(r.data) != expectedLength {
+		t.Errorf("RecordValue: expected length: %v, actual: %v", expectedLength, len(r.data))
+	}
+
+}
```

Comment:
Can you add a similar test case where window == the intitial length, and then another item is added.  An assertion about the order of the internal data array wouldn't go awry.